examples/purchase-tester-java/src/main/java/com/revenuecat/purchasetester/java/CatsActivity.java

```diff
@@ -30,31 +30,23 @@ protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_cats);
 
-        findViewById(R.id.go_premium).setOnClickListener(new View.OnClickListener() {
-            @Override
-            public void onClick(View v) {
-                finish();
-            }
-        });
+        findViewById(R.id.go_premium).setOnClickListener(v -> finish());
```

Comment:
Love it! Can this be? I am actually not sure

```suggestion
        findViewById(R.id.go_premium).setOnClickListener(::finish);
```

Reply:
unfortunately no. it does look kinda weird after being used to kotlin 😅